06-reactJs/components/form.js

```diff
@@ -0,0 +1,30 @@
+var React = require('react');
+var _ = require('lodash');
+
+var Input = require('input')
+
+var FormGroup = React.createClass({
+
+    getInitialState: function () {
+        inputText:'';
+    }
+
+    render: function () {
+        <form>
+            <Input {...this.getInputProps()} />
+        </form>
+    }
+
+    getInputProps: function () {
+        return {
+            value: this.state.inputText,
+            onChange: this.handleInputChange()
+        }
+    }
+
+    handleInputChange: function (event) {
+        this.setState({event.target.value})
```

Comment:
This is not right, you need to set a key value to generate a plain object. Be aware of this...